### PR TITLE
139 christmas tree instructions -Combined PR

### LINKED ISSUE #139: As someone interested in obtaining a Christmas tree permit, I should see precise language about tree cutting limits, so that I understand the limit on the number of trees I can cut
## Notes
Under Cost it reads, “Christmas trees cost $5.00 each, and a household is allowed to cut up to 3 trees per permit.” 

The “per permit” language implies there is no limit on how many permits a person or household can obtain.

It should read, “Christmas trees cost $5.00 each, and a household is allowed to cut up to 3 trees.” 

## Acceptance Criteria
- [ ] There are no instances of "up to X trees per permit" in the app (including on the permit itself, and in the email)

## Definition of Done
- [ ] Code unit / e2e tested and comply with QWASP
- [ ] No new security vulnerabilities
- [ ] Internal team code review
- [ ] Documentation / readme.md / CircleCI updated
- [ ] Docker updated
- [ ] Compare finished design with mockup
- [ ] Usability tested

### PR DESCRIPTION
**Pull request by lauraGgit**
Partially fixes #139 as there are 5 pages to change

## Summary
Resolves Issue #[139]
Updates the text for clarity of how many trees.

## Impacted Areas of the Site
- Christmas trees
- wiki

## Optional Screenshots
NA
## This pull request changes...
- [ ] change app form for xmas
- [ ] change app rules (also on email)

## This pull request is ready to merge when...
- [x] Tests have been updated (and all tests are passing)
- [ ] This code has been reviewed by someone other than the original author
- [x] The change has been documented
  - [x] Associated OpenAPI documentation has been updated

### CODE REVIEW

File: wiki/christmas-trees/usability-testing/_Sidebar.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 **Schedule**
 
-[Usability testing research schedule](https://github.com/nciinc/fs-permit-platform/wiki/Usability-testing-research-schedule)
+[Usability testing research schedule](https://github.com/nciinc/fs-open-forest-platform/wiki/Usability-testing-research-schedule)
```
Comment (adborden) [inline wiki/christmas-trees/usability-testing/_Sidebar.md]: These links don't seem to work, I don't think NCI's fork was renamed.

Comment (lauraGgit) [reply wiki/christmas-trees/usability-testing/_Sidebar.md]: good catch, just updated the refs to 18F

Comment (lauraGgit) [reply wiki/christmas-trees/usability-testing/_Sidebar.md]: @adborden and so ready for additional review

Comment (adborden) [reply wiki/christmas-trees/usability-testing/_Sidebar.md]: These links still don't work. Should we migrate the NCI wiki over to this repo? I think that would be the easiest. Right now there's only one page on the wiki: https://github.com/18F/fs-open-forest-platform/wiki

**Comment by lauraGgit** [comment]
@adborden ok, think i finally resolved the wiki ref issues.

**Pull request merged**